features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/source/src/app/components/Apis/Details/BasicInfo.js

```diff
@@ -58,6 +58,7 @@ class BasicInfo extends Component {
         };
         this.api_uuid = this.props.uuid;
         this.logChange = this.logChange.bind(this);
+       // this.handleStarRatingUpdate = this.handleStarRatingUpdate(this);
```

Comment:
Shall we remove the commented code line?

Reply:
removed